README.md

```diff
@@ -4,6 +4,81 @@ This repository contains:
+
+   ```
+   EMISSION_MANAGER.configureAssets([{
+
+     emissionPerSecond: The emission per second following rewards unit decimals.
+
+     totalSupply: The total supply of the asset to incentivize. This should be kept as 0 as the Emissions Manager will fill this up.
+
+     distributionEnd: The end of the distribution of rewards (in seconds).
+
+     asset: The asset for which rewards should be given. Should be the address of the aave aToken (for deposit) or debtToken (for borrow).
+            In case where the asset for reward is for debt token please put the address of stable debt token for rewards in stable borrow mode
+            and address of variable debt token for rewards in variable borrow mode.
+
+     reward: The reward token address to be used for Liquidity Mining for the asset.
+
+     transferStrategy: The address of transfer strategy contract.
+
+     rewardOracle: The Chainlink Aggregator compatible Price Oracle of the reward.
+
+   }])
+   ```
+
+Below is an example with the pseudo code to activate Liquidity Mining for the variable borrow of `wMatic` with `MaticX` as the reward token for the total amount of `60,000` `MaticX` for the total duration of `6 months`. For a more detailed explanation checkout this [test](./tests/EmissionTestMATICXPolygon.t.sol).
+
+1. Make sure the Rewards Vault has sufficient balance of the MaticX token.
+
+   ```
+   IERC20(MATIC_X_ADDRESS).balanceOf(REWARDS_VAULT) > 60000 *1e18
+   ```
+
+2. Do an ERC-20 approve from the MaticX token from the Rewards Vault to the transfer strategy contract for the total amount.
+
+   ```
+   IERC20(MATIC_X_ADDRESS).approve(TRANSFER_STRATEGY_ADDRESS, 60000 *1e18);
+   ```
+
+3. Configure the Liquidity Mining emissions on the Emission Manager contract.
+
+   ```
+   EMISSION_MANAGER.configureAssets([{
+
+     emissionPerSecond: 60000 * 1e18 / (180 days in seconds)
+
+     totalSupply: 0
+
+     distributionEnd: current timestamp + (180 days in seconds)
+
+     asset: Aave Variable Debt Token of wMatic // 0x4a1c3aD6Ed28a636ee1751C69071f6be75DEb8B8
+
+     reward: MaticX Token address // 0xfa68FB4628DFF1028CFEc22b4162FCcd0d45efb6
+
+     transferStrategy: ITransferStrategyBase(STRATEGY_ADDRESS) // 0x53F57eAAD604307889D87b747Fc67ea9DE430B01
+
+     rewardOracle: IEACAggregatorProxy(MaticX_ORACLE_ADDRESS) // 0x5d37E4b374E6907de8Fc7fb33EE3b0af403C7403
+
+   }])
+   ```
+
+Awesome! so liquidity mining has been succefully configured now.
+
+After the Liquidity Mining has been set, we can also configure later on to increase or decrease the rewards (emissions per second) and to change the end date for liquidity mining. More info on this below.
 
 ## How to modify emissions of the LM program?
```

Comment:
Why this is needed, if the correct way is the one you describe on `How to configure emissions after the LM program has been created?`?